src/main/java/com/flightstats/hub/spoke/SpokeTtlEnforcerService.java

```diff
@@ -0,0 +1,31 @@
+package com.flightstats.hub.spoke;
+
+import com.google.common.util.concurrent.AbstractScheduledService;
+import lombok.extern.slf4j.Slf4j;
+
+import javax.inject.Inject;
+import java.util.concurrent.TimeUnit;
+
+@Slf4j
+public class SpokeTtlEnforcerService extends AbstractScheduledService {
+
```

Comment:
Thanks for abstracting this out from a nested class. Most recently when this has been done, classes patterned like this that extend Abstract*Service we have used Lifecycle as a naming scheme that is a little more specifically semantic in this context. I think this should be renamed SpokeTtlEnforcerLifecycle for consistency